src/latexify/transformers/function_expander.py

```diff
@@ -39,6 +39,71 @@ def visit_Call(self, node: ast.Call) -> ast.AST:
         return node
 
 
+def _atan2_expander(function_expander: FunctionExpander, node: ast.Call) -> ast.AST:
+    if len(node.args) != 2:
+        raise exceptions.LatexifyNotSupportedError(
+            "FunctionExpander only supports expanding 'atan2' with two arguments"
+        )
```

Comment:
Should raise `LatexifySyntaxError` because `atan2` takes exactly 2 arguments.
```suggestion
        raise exceptions.LatexifySyntaxError(
            "Incorrect number of arguments for atan2."
        )
```

Ditto for other functions.

I also guess it is better to introduce a validator function to check if the number of arguments is correct or not:

```python
def check_num_args(node: ast.Call, nargs: int) -> None:
    if len(node.args) != nargs:
        fn_name = ast_utils.extract_function_name_or_none(node)
        raise exceptions.LatexifySyntaxError(
            f"Incorrect number of arguments for {fn_name}."
            f" expected: {nargs}, but got {len(node.args)}"
```